WordPress/src/main/java/org/wordpress/android/ui/blaze/BlazeFeatureUtils.kt

```diff
@@ -46,7 +46,9 @@ class BlazeFeatureUtils @Inject constructor(
     }
 
     fun isSiteBlazeEligible(siteModel: SiteModel): Boolean {
-        return siteModel.canBlaze && siteModel.isAdmin &&
+        return siteModel.canBlaze != null &&
+                siteModel.canBlaze &&
+                siteModel.isAdmin &&
```

Comment:
@AjeshRPai - ❓ I thought we didn't need the isAdmin check, but it's quite likely that I am mistaken.

Reply:
Hey @zwarm 

I think We need the Admin check, checked IOS PR - [they have same logic](https://github.com/wordpress-mobile/WordPress-iOS/pull/20916/files). I think can_blaze returns true for contributor role as well, don't remember but I think I checked that.